src/EnergyPlus/UnitarySystem.hh

```diff
@@ -486,6 +486,7 @@ namespace UnitarySystems {
         bool m_ZoneCompFlag = true;
         std::string m_AvailManagerListName;
         int m_EquipCompNum = 0; // 1-based index of this parent type for specific equipment type processing
+        bool reportVariablesAreSetup = false;
```

Comment:
Where is this new variable used? I can't find it anywhere.

Reply:
@rraustad No need to use this variable. Remove it.